package.json

```diff
@@ -86,6 +86,7 @@
     "electron-squirrel-startup": "^1.0.0",
     "keymirror": "0.1.1",
     "electron-context-menu": "^0.8.0",
+    "windows-shortcuts": "^0.1.4",
```

Comment:
Hi Lynn. 
node-auto-launch seems to be much more elegant than windows-shortcuts. I´m using this, will commit the change in some minutes.